crates/ruma-events/src/room/message.rs

```diff
@@ -690,3 +758,242 @@ pub struct CustomEventContent {
+            formatdoc!(
+                "
+                <mx-reply>
+                    <blockquote>
+                        <a href=\"https://matrix.to/#/{room_id}/{event_id}\">In reply to</a>
+                        <a href=\"https://matrix.to/#/{sender}\">{sender}</a>
+                        <br />
+                        {body}
+                    </blockquote>
+                </mx-reply>",
+                room_id = original_message.room_id,
+                event_id = original_message.event_id,
+                sender = original_message.sender,
+                body = formatted_or_plain_body(&content.formatted, &content.body),
+            )
+        }
+        MessageType::Video(_) => {
+            formatdoc!(
+                "
+                <mx-reply>
+                    <blockquote>
+                        <a href=\"https://matrix.to/#/{room_id}/{event_id}\">In reply to</a>
+                        <a href=\"https://matrix.to/#/{sender}\">{sender}</a>
+                        <br />
+                        sent a video.
+                    </blockquote>
+                </mx-reply>",
+                room_id = original_message.room_id,
+                event_id = original_message.event_id,
+                sender = original_message.sender,
+            )
+        }
+        MessageType::_Custom(content) => {
+            formatdoc!(
+                "
+                <mx-reply>
+                    <blockquote>
+                        <a href=\"https://matrix.to/#/{room_id}/{event_id}\">In reply to</a>
+                        <a href=\"https://matrix.to/#/{sender}\">{sender}</a>
+                        <br />
+                        {body}
+                    </blockquote>
+                </mx-reply>",
+                room_id = original_message.room_id,
+                event_id = original_message.event_id,
+                sender = original_message.sender,
+                body = content.data["body"].as_str().unwrap_or(""),
+            )
+        }
+        #[cfg(feature = "unstable-pre-spec")]
+        MessageType::VerificationRequest(content) => {
+            formatdoc!(
+                "
+                <mx-reply>
+                    <blockquote>
+                        <a href=\"https://matrix.to/#/{server_name}/{event_id}\">In reply to</a>
+                        <a href=\"https://matrix.to/#/{sender}\">{sender}</a>
+                        <br />
+                        {body}
+                    </blockquote>
+                </mx-reply>",
+                server_name = original_message.room_id,
+                event_id = original_message.event_id,
+                sender = original_message.sender,
+                body = content.body,
+            )
+        }
+    }
+}
+
+fn formatted_or_plain_body<'a>(formatted: &'a Option<FormattedBody>, body: &'a str) -> &'a str {
+    if let Some(formatted_body) = formatted {
+        &formatted_body.body
+    } else {
+        &body
```

Comment:
As per clippy message:

```suggestion
        body
```

Reply:
This should be more obvious through a failing CI job, something seems broken there (see #562).